### PR TITLE
rework ui

### PR DESCRIPTION
**Pull request by ZhekehZ**

### CODE REVIEW

File: src/main/java/ru/itmo/roguelike/characters/inventory/Usable.java [hunk 1]
```diff
@@ -58,50 +61,39 @@ static Usable readFromFile(DataInputStream input, Player player) throws IOExcept
...
 
-        @Override
-        public boolean equals(Object o) {
-            if (!(o instanceof Sort)) return false;
-            return word.equals(((Sort) o).word);
-        }
+    static Usable readFromFile(DataInputStream input, Player player) throws IOException {
+        String sort = String.valueOf(new char[]{input.readChar(), input.readChar(), input.readChar()});
+        UsableCreator creator = creators.get(sort);
+        return creator.create(input, player);
```
Comment (HarmlessEvil) [inline src/main/java/ru/itmo/roguelike/characters/inventory/Usable.java:97]: Is all of this has package private access modifier intentionally?

File: src/main/java/ru/itmo/roguelike/items/MedKit.java [hunk 2]
```diff
@@ -53,13 +45,19 @@ public void use(@NotNull Actor actor) {
     }
 
     @Override
-    public Sort getSign() {
-        return MEDKIT_SORT;
+    public String getSort() {
+        return SORT;
+    }
+
+    public static MedKit fromFile(DataInputStream inputStream, Player p) throws IOException {
```
Comment (HarmlessEvil) [inline src/main/java/ru/itmo/roguelike/items/MedKit.java:52]: This definitely needs a comment: user should know, that file has to be at correct position, that contains MedKit's int value

**Review by HarmlessEvil — COMMENTED** [review]
About font: it's better to leave original font name, rather than just "font.ttf"

File: src/main/java/ru/itmo/roguelike/characters/inventory/Usable.java (hunk 1 above)
Comment (ZhekehZ) [reply src/main/java/ru/itmo/roguelike/characters/inventory/Usable.java:97]: Это ж интерфейс

File: src/main/java/ru/itmo/roguelike/items/MedKit.java (hunk 2 above)
Comment (ZhekehZ) [reply src/main/java/ru/itmo/roguelike/items/MedKit.java:52]: Го после мержа добавим разом все комменты?

File: src/main/java/ru/itmo/roguelike/characters/inventory/Usable.java (hunk 1 above)
Comment (HarmlessEvil) [reply src/main/java/ru/itmo/roguelike/characters/inventory/Usable.java:97]: Не заметил)

File: src/main/java/ru/itmo/roguelike/items/MedKit.java (hunk 2 above)
Comment (HarmlessEvil) [reply src/main/java/ru/itmo/roguelike/items/MedKit.java:52]: Оки

**Comment by HarmlessEvil** [comment]
You forgot `feature/` prefix in branch name)

**Pull request merged**